README.md

```diff
@@ -31,3 +31,53 @@ For people who are trying to modify the image stack, here are some scenarios and
 4. If the workflow failed, go to the action details page and debug the issue. Push a new change by amending the first commit `git add . && git commit --amend`.
 5. If the workflow completed successfully (a green check to the left of the commit), you can now safely merge the pull request to the `main` branch.
 6. The workflow will now run again and push the images to Dockerhub.
+
+
+### Overview of the Repository
+We Use github workflow to builds new images if their is any change in the images or addtional images are added.<br>
+.github:-<br>
+        The main.yml file contains the steps to build the image<br>
+            - Initial download required for the images are installed<br>
+            - Doit module is used to run the tasks<br>
+                1) install all the requirments for the images<br>

```

Comment:
Try using markdown number scheme. Instead of 1) use 1. 2. etc.

Reply:
changed it markdown